intranet/apps/polls/models.py

```diff
@@ -94,6 +94,32 @@ def get_users_voted(self):
                 users = list(q.get_users_voted())
         return users
 
+    def get_num_eligible_voters(self):
+        groups = DjangoGroup.objects.none()
+        if self.groups.exists():
+            q_objects = Q()
+            for g in self.groups.all():
+                q_objects.add(Q(name=g), Q.OR)
+            groups = DjangoGroup.objects.filter(q_objects)
+            users = get_user_model().objects.none()
+            for g in groups:
+                users = users.union(g.user_set.all())
+            return users.count()
+        else:
+            return get_user_model().objects.count() - 1
+
+    def get_percentage_voted(self, voted, able):
+        percentage = voted / able
```

Comment:
If able is 0, this raises a divide-by-zero error. I would recommend using a `try`-`except` block.

```python
    percentage = 0
    try:
        percentage = voted / able
    except ZeroDivisionError:
        pass
```